pages/documentation/showcase.md

```diff
@@ -22,20 +22,24 @@ If your project is currently using USWDS and you would like to see it included i
 - [18F Before you ship](https://before-you-ship.18f.gov/)
 - [18F Front End Guide](https://frontend.18f.gov/)
 - [Accessibility for Teams](https://accessibility.digital.gov/)
-- [Advanced Distributed Learning (ADL) Initiative](https://adlnet.gov)
-- [Agricultural Marketing Service - USDA](https://www.ams.usda.gov/)
-- [Agricultural Research Service - USDA](https://www.ars.usda.gov/)
-- [Scientific Discoveries - Agricultural Research Service - USDA](https://scientificdiscoveries.ars.usda.gov/)
-- [America’s Seed Fund - National Science Foundation](https://seedfund.nsf.gov/)
+- [Advanced Distributed Learning Initiative](https://adlnet.gov)
+- [Agricultural Marketing Service (USDA)](https://www.ams.usda.gov/)
+- [Agricultural Research Service (USDA)](https://www.ars.usda.gov/)
+- [Scientific Discoveries (Agricultural Research Service - USDA)](https://scientificdiscoveries.ars.usda.gov/)
```

Comment:
Should this be moved to maintain alphabetical order?

Reply:
Yes, please, @amyleadem . I will note this in the audit spreadsheet I have started. Do I need to "resolve conversation" to acknowledge this, or just comment and you'll resolve once you've updated?

Reply:
I'll leave a note here once I update it. If the update looks good to you, you can resolve the conversation.

Reply:
I updated the order of this in [9ec848c](https://github.com/uswds/uswds-site/pull/2272/commits/9ec848c8587323c5b16f5a8c45bed06395f303bc). Let me know if you need any changes!